gooddata-java/src/main/java/com/gooddata/sdk/service/GoodData.java

```diff
@@ -355,4 +356,13 @@ public ExecuteAfmService getExecuteAfmService() {
     public LcmService getLcmService() {
         return services.getLcmService();
     }
+
+    /**
+     * Get initialized service for config management
```

Comment:
Please use the latest naming in this javadoc

Reply:
Changed